### DISCUSSION TITLE
Pouvoir changer les noms de "Leader"/"Traître"

### DISCUSSION THREAD

**Discussion by ElethornGM** [fibreville/minotaure_php#68] [Boite à idées]
Pouvoir changer le nom du "Leader" / "Traitre" (un peu comme on peut changer esprit/corps)

Par exemple, pouvoir appeler le Leader "Capitaine" et le Traître "Conseiller"...

**Comment by uurshin** [comment]
Envisageable et envisagé pour une prochaine version.
